docs/CONTRIBUTING.md

```diff
@@ -76,8 +76,7 @@ as well.
 In order to run the frontend, you will need to have installed:
 
 - [node](https://nodejs.org/en/) >= 6.10.0
-- [npm](https://www.npmjs.com/get-npm) >= 3.10.10
-- [yarn](https://yarnpkg.com/en/docs/install) >= 0.23.2
+- [npm](https://www.npmjs.com/get-npm) >= 4.0.0
```

Comment:
it works fine with npm4 or greater, so that is what is listed. npm 4 will ignore the lock file but that is not mission critical to being able to develop the website.

npm 4 is the lowest possible version of npm that still has maintenance, which is why i picked it.